diff --git a/native/python/src/fairseq2n/__init__.py b/native/python/src/fairseq2n/__init__.py
index 381bb3d..7adce7c 100644
--- a/native/python/src/fairseq2n/__init__.py
+++ b/native/python/src/fairseq2n/__init__.py
@@ -3,8 +3,8 @@
 #
 # This source code is licensed under the BSD-style license found in the
 # LICENSE file in the root directory of this source tree.
-
 from __future__ import annotations
+import torch
 
 __version__ = "0.4.5"
 
@@ -166,7 +166,9 @@ def _load_shared_library(lib_name: str) -> CDLL | None:
 
 # We load shared libraries that we depend on using our own extended lookup logic
 # since they might be located in non-default locations.
-_load_shared_libraries()
+
+# Charles : No need as libraries are rpathed, plus this does not even work properly.
+#_load_shared_libraries()
 
 
 def _check_torch_version() -> None:
