diff -ur a/jaxtyping/__init__.py b/jaxtyping/__init__.py
--- a/jaxtyping/__init__.py	2025-08-11 15:53:18.371605552 +0000
+++ b/jaxtyping/__init__.py	2025-08-11 15:54:31.954404381 +0000
@@ -22,6 +22,7 @@
 import typing
 import warnings
 from typing import TypeAlias, Union
+import re
 
 from ._array_types import (
     AbstractArray as AbstractArray,
@@ -225,7 +226,7 @@
     except importlib.metadata.PackageNotFoundError:
         pass
     else:
-        major, minor, patch = eqx_version.split(".")
+        major, minor, patch = re.split(r'[.+]', eqx_version)[:3]
         equinox_version = (int(major), int(minor), int(patch))
         if equinox_version < (0, 11, 0):
             warnings.warn(
