diff --git a/jaxtyping/__init__.py b/jaxtyping/__init__.py
index 991f1b1..3abc6c5 100644
--- a/jaxtyping/__init__.py
+++ b/jaxtyping/__init__.py
@@ -220,7 +220,7 @@ if check_equinox_version:
     except importlib.metadata.PackageNotFoundError:
         pass
     else:
-        major, minor, patch = eqx_version.split(".")
+        major, minor, patch = eqx_version.partition("+")[0].split(".")
         equinox_version = (int(major), int(minor), int(patch))
         if equinox_version < (0, 11, 0):
             warnings.warn(
