diff --git a/ffi/Makefile.linux b/ffi/Makefile.linux
index 334e9ff..99f360c 100644
--- a/ffi/Makefile.linux
+++ b/ffi/Makefile.linux
@@ -19,7 +19,7 @@ all: $(OUTPUT)
 $(OUTPUT): $(SRC) $(INCLUDE)
 	# static-libstdc++ avoids runtime dependencies on a
 	# particular libstdc++ version.
-	$(CXX) $(CXX_STATIC_LINK) -shared $(CXXFLAGS) $(SRC) -o $(OUTPUT) $(LDFLAGS) $(LIBS)
+	$(CXX) $(CXX_STATIC_LINK) -shared -fPIC $(CXXFLAGS) $(SRC) -o $(OUTPUT) $(LDFLAGS) $(LIBS)
 
 clean:
 	rm -rf test $(OUTPUT)
