diff --git a/CMakeLists.txt b/CMakeLists.txt
index 9506300..d295916 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -45,7 +45,9 @@ if(CMAKE_CUDA_COMPILER)
 
     enable_language(CUDA)
     set(CUDA_OPTS "-Wreorder -Xcompiler -fPIC -Xcompiler -Wall -Xcompiler -fopenmp -Xptxas -dlcm=ca --cudart static --relocatable-device-code=false")
-    string(APPEND CUDA_OPTS " -gencode arch=compute_70,code=sm_70 -gencode arch=compute_75,code=sm_75 -gencode arch=compute_80,code=sm_80 -gencode arch=compute_86,code=sm_86")
+    if(NOT DEFINED CMAKE_CUDA_ARCHITECTURES)
+            set(CMAKE_CUDA_ARCHITECTURES 70 75 80 86)
+    endif()
     if(CMAKE_BUILD_TYPE MATCHES Debug)
         string(APPEND CUDA_OPTS " -G")
     else()
