diff --git a/setup.py b/setup.py
index 9d1b069..24b6f88 100644
--- a/setup.py
+++ b/setup.py
@@ -21,12 +21,7 @@ https://github.com/sanguinariojoe/pip-openexr/issues
 """
 
 
-print("Looking for libOpenEXR...")
-if platform.system() == "Linux" and system("ldconfig -p | grep libOpenEXR"):
-    # There is no libOpenEXR, probably an old version of OpenEXR
-    libraries=['Iex', 'Half', 'Imath', 'IlmImf', 'z']
-else:
-    libraries=['Iex', 'OpenEXR', 'z']
+libraries=['Iex', 'OpenEXR', 'z']
 
 extra_compile_args = ['-g', '-DVERSION="%s"' % VERSION]
 if platform.system() == 'Darwin':
@@ -44,15 +39,6 @@ setup(name='OpenEXR',
   ext_modules=[ 
     Extension('OpenEXR',
               ['OpenEXR.cpp'],
-              include_dirs=['/usr/include/OpenEXR',
-                            '/usr/local/include/OpenEXR',
-                            '/opt/local/include/OpenEXR',
-                            '/usr/include/Imath',
-                            '/usr/local/include/Imath',
-                            '/opt/local/include/Imath'],
-              library_dirs=['/usr/lib',
-                            '/usr/local/lib',
-                            '/opt/local/lib'],
               libraries=libraries,
               extra_compile_args=extra_compile_args)
   ],
