diff --git a/python/setup.py b/python/setup.py
index 029c571..f70597b 100644
--- a/python/setup.py
+++ b/python/setup.py
@@ -62,17 +62,6 @@ class bdist_wheel(_bdist_wheel):
         _bdist_wheel.finalize_options(self)
         self.root_is_pure = False
 
-    # https://github.com/Yelp/dumb-init/blob/57f7eebef694d780c1013acd410f2f0d3c79f6c6/setup.py#L25
-    def get_tag(self):
-        python, abi, plat = _bdist_wheel.get_tag(self)
-        if plat[:5] == "linux":
-            libc = ctypes.CDLL("libc.so.6")
-            libc.gnu_get_libc_version.restype = ctypes.c_char_p
-            GLIBC_VER = libc.gnu_get_libc_version().decode("utf8").split(".")
-            plat = f"manylinux_{GLIBC_VER[0]}_{GLIBC_VER[1]}{plat[5:]}"
-        return python, abi, plat
-
-
 cmdclass["bdist_wheel"] = bdist_wheel
 
 
