diff --git a/3rdparty/stdgpu/stdgpu.cmake b/3rdparty/stdgpu/stdgpu.cmake
index a725f2280..7d99ab9f5 100644
--- a/3rdparty/stdgpu/stdgpu.cmake
+++ b/3rdparty/stdgpu/stdgpu.cmake
@@ -24,10 +24,10 @@ ExternalProject_Add(
     CMAKE_CACHE_ARGS    # Lists must be passed via CMAKE_CACHE_ARGS
         -DCMAKE_CUDA_ARCHITECTURES:STRING=${CMAKE_CUDA_ARCHITECTURES}
     BUILD_BYPRODUCTS
-        <INSTALL_DIR>/lib/${CMAKE_STATIC_LIBRARY_PREFIX}stdgpu${CMAKE_STATIC_LIBRARY_SUFFIX}
+        <INSTALL_DIR>/lib64/${CMAKE_STATIC_LIBRARY_PREFIX}stdgpu${CMAKE_STATIC_LIBRARY_SUFFIX}
 )
 
 ExternalProject_Get_Property(ext_stdgpu INSTALL_DIR)
 set(STDGPU_INCLUDE_DIRS ${INSTALL_DIR}/include/) # "/" is critical.
-set(STDGPU_LIB_DIR ${INSTALL_DIR}/lib)
+set(STDGPU_LIB_DIR ${INSTALL_DIR}/lib64)
 set(STDGPU_LIBRARIES stdgpu)
