diff --git a/src/rich_click/_compat_click.py b/src/rich_click/_compat_click.py
index 47c4e9e..1b0a44f 100644
--- a/src/rich_click/_compat_click.py
+++ b/src/rich_click/_compat_click.py
@@ -8,7 +8,7 @@ import click
 try:
     with warnings.catch_warnings():
         warnings.simplefilter(category=DeprecationWarning, action="ignore")
-        click_version = click.__version__
+        click_version: str = click.__version__  # type: ignore[assignment]
 except Exception:
     # Click 9+ deprecated __version__, so all these checks must necessarily be False if __version__ doesn't exist.
     CLICK_IS_BEFORE_VERSION_821 = False
@@ -16,6 +16,7 @@ except Exception:
     CLICK_IS_BEFORE_VERSION_9X = False
     CLICK_IS_VERSION_80 = False
 else:
+    click_version, _, _ = click_version.partition("+")  # Partition on local version, drop anything after `+`
     _major = int(click_version.split(".")[0])  # type: ignore[attr-defined,unused-ignore]
     _minor = int(click_version.split(".")[1])  # type: ignore[attr-defined,unused-ignore]
     _patch = int(click_version.split(".")[2])  # type: ignore[attr-defined,unused-ignore]
